pkgs/development/python-modules/uri-template/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "uri_template";
+  version = "1.1.0";
+  owner = "plinss";
```

Comment:
owner should be in src

Reply:
I thought I can just add my own variables here, which "owner" is meant to be